consensus/hotstuff/verifier.go

```diff
@@ -41,5 +41,5 @@ type Verifier interface {
 	// * signature.ErrInvalidFormat if the signature has an incompatible format.
 	// * unexpected errors should be treated as symptoms of bugs or uncovered
 	//   edge cases in the logic (i.e. as fatal)
-	VerifyQC(voters flow.IdentityList, sigData []byte, block *model.Block) (bool, error)
+	VerifyQC(voters flow.IdentityList, sigData []byte, block *model.Block) error
```

Comment:
It would be good tp update the go doc too to mirror the new API

Reply:
Good point. Updated here
https://github.com/onflow/flow-go/pull/1779/files#diff-808ac5bc91604a4ce38e9e740a6efe7ef6701b1e935d4bd1499d6ae1bf9ad406R35